app/cdap/components/ConnectionList/index.tsx

```diff
@@ -234,39 +237,49 @@ export default function ConnectionList() {
                     variant="body2"
                     className={classes.headerLabel}
                     ref={(element) => {
-                      refs.current[index] = element;
+                      refs.current[tabIndex] = element;
                     }}
                   >
-                    {dataForTabs[index - 1].selectedTab}
+                    {dataForTabs[tabIndex - 1].selectedTab}
                   </Typography>
                 </Box>
               );
           }
           return (
-            <Box className={classes.tabsContainerWithHeader}>
+            <Box
+              className={classes.tabsContainerWithHeader}
+              data-testid={`connection-tabs-column-${tabIndex}`}
```

Comment:
can we use some label or name of the connection tab rather than index in the data-testid